lang/test/org/partiql/lang/eval/visitors/VisitorTransformTestBase.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package org.partiql.lang.eval.visitors
+
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.partiql.lang.ast.toAstStatement
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.syntax.SqlParserTestBase
+
+abstract class VisitorTransformTestBase : SqlParserTestBase() {
```

Comment:
This class performs exactly the same function as `RewriterTestBase` but is shorter since PIG-generated domain models are nicer to work with than `ExprNode`.

Reply:
Note to self: add `kdoc`.